### ISSUE TITLE
LR_SGD purpose

### ISSUE THREAD

**Issue by ArturPrzybysz** [noelcodella/tripletloss-keras-tensorflow#3]
Hey, thanks for triple loss example!
Could you explain why LR_SGD is used? Could I use any Keras-provided optimizers instead?
Greetings

**Comment by noelcodella** [comment]
It's just an example of how to adjust the learning rate across the layers of the network. You can use any optimizer you like.
